src/platform/backends/lxd/lxd_virtual_machine.cpp

```diff
@@ -459,3 +470,15 @@ void mp::LXDVirtualMachine::resize_disk(const MemorySize& new_size)
     QJsonObject patch_json{{"devices", QJsonObject{{"root", root_json}}}};
     lxd_request(manager, "PATCH", url(), patch_json);
 }
+
+std::unique_ptr<multipass::MountHandler>
+mp::LXDVirtualMachine::make_native_mount_handler(const SSHKeyProvider* ssh_key_provider, const std::string& target,
+                                                 const VMMount& mount)
+{
+    if (!is_default_Gid_Uid(mount))
+    {
+        throw std::runtime_error("LXD native mount does not accept user provided gid or uid.");
```

Comment:
```suggestion
        throw std::runtime_error("LXD native mount does not accept custom gid or uid.");
```

This class does not know where IDs come from. We know they come from the user but, as far as this class is concerned, they could also come from a custom client or whatever.